src/main/java/net/mcreator/ui/modgui/codeviewer/ModElementCodeViewer.java

```diff
@@ -104,9 +104,9 @@ public void registerUI(JComponent container) {
 		}
 	}
 
-	private synchronized void reload() {
+	private void reload() {
 		if (isVisible() && !updateRunning) {
-			new Thread(() -> {
+			synchronized (this) {

```

Comment:
Removing thread there will make reload run on UI thread which is not ok

Reply:
I see. I thought it may be useful now, but OKay, I'll leave it for #2010 so I have more time to find a proper solution